### ISSUE TITLE
Seems not to work

### ISSUE THREAD

**Issue by Jey-Cee** [NodeBB/nodebb-plugin-webhooks#3]

**Comment by psychobunny** [comment]
What doesn't work? :) 

If there's an error please post what you see?

**Comment by Jey-Cee** [comment]
Sorry, for the missing information. I didn't send the bug report at all, at least I thought I did ;-)
My Problem is there is no issue in the output.log, everything seems to be ok, but the webhook isn't fired at all.

I've tried to build my own server application with a webhook route, as i'm new to this topic and not sure if it works corret i tried to use Discord. There we have already a channel where are new topics pushed to it by the discord notifaction plugin.
So it is save that this will work.

I tried to use action:**.**, action:post.save and other action hooks. In System > Webhooks the only hook that is registered by the plugin is static:app.load.

Forum was restarted and also rebuilded, without success.

Nodebb version: 1.16.1 & 1.16.0
Plugin version: 1.0.9

Is there a log for the plugin itself?
